ledger/complete/wal/checkpoint_v6_reader.go

```diff
@@ -631,6 +634,60 @@ func readTopLevelTries(dir string, fileName string, subtrieNodes [][]*node.Node,
 	return tries, nil
 }
 
+func readTriesRootHash(logger zerolog.Logger, dir string, fileName string) (
+	trieRoots []ledger.RootHash,
```

Comment:
how many tries does a normal checkpoint have? can this be used to get the root hash for the whole trie in the checkpoint?

Reply:
it could be from 1 to 500.